README.md

```diff
@@ -1,27 +1,33 @@
 # Zero Forcing Number
-This program calculates the zero forcing number (and set) of any graph. The zero forcing number is defined as follows: "A subset S of initially infected vertices of a graph G is called forcing if we can infect the entire graph by iteratively applying the following process. At each step, any infected vertex which has a unique uninfected neighbour, infects this neighbour. The forcing number of G is the minimum cardinality of a forcing set in G," according to [Kalinowski, Kamčev, and Sudakov](https://arxiv.org/abs/1705.10391). 
+This program calculates the zero forcing number (and set) of any graph. The zero forcing number is defined as follows:
+>A subset S of initially infected vertices of a graph G is called forcing if we can infect the entire graph by iteratively applying the following process. At each step, any infected vertex which has a unique uninfected neighbour, infects this neighbour. The forcing number of G is the minimum cardinality of a forcing set in G.
+- [Kalinowski, Kamčev, Sudakov](https://arxiv.org/abs/1705.10391) 
 
 ---
 
-## Usage:
-### Build:
-`sage --python3 setup.py build_ext`
+## Usage
+### Install
+`sage -pip install .`
+
+### Test
+`sage -pip install zeroforcing[test]`
+* Installs the dependencies for testing
+
+`sage --python3 setup.py wavefront`
+* Builds the wavefront code for verification during tests
+
+`sage -pip install .`
+* Installs the wavefront code we just built
 
-### Test:
 `sage --python3 -m pytest [-x]`
 * `-x` flag makes pytest stop after the first failure
 * `-h` flag will show a section called `Zero forcing options:`
 
-### Clean:
-`sage --python3 setup.py clean`
-
-### Help:
-`sage --python3 setup.py -h`
-(You can also use the `-h` flag in subcommands. i.e. `sage --python3 setup.py build_ext -h`
+### Import
+`from zeroforcing.metagraph import zero_forcing_set`
```

Comment:
There is also zero_forcing_number (and there will be other functions in the future).  Is there a way to tell it to import _everything_?  It seems that ``from zeroforcing.metagraph import *`` works, but seems a little clunky.  For one thing, to hide the implementation completely, we'd want to avoid any mention of "metagraph".  If we add an ``__init.py__`` to ``zeroforcing`` itself, can we just have that import whatever we want upon ``import zeroforcing``? :)